staging/operator-lifecycle-manager/pkg/package-server/server/server.go

```diff
@@ -16,14 +16,29 @@ import (
 	"k8s.io/client-go/kubernetes"
 	"k8s.io/client-go/rest"
 	"k8s.io/client-go/tools/clientcmd"
+	"k8s.io/client-go/util/workqueue"
 
+	operatorsv1 "github.com/operator-framework/api/pkg/operators/v1"
 	"github.com/operator-framework/operator-lifecycle-manager/pkg/api/client"
+	"github.com/operator-framework/operator-lifecycle-manager/pkg/api/client/clientset/versioned"
```

Comment:
nit: it's nice to alias this to `olmclientset` or something similar, `versioned` is a nondescript name